osu.Game/Overlays/Mods/ModSection.cs

```diff
@@ -50,6 +50,10 @@ public IEnumerable<Mod> Mods
 
                 ButtonsContainer.Children = modContainers;
                 buttons = modContainers.OfType<ModButton>().ToArray();
+
+                var expand = value.Any();
+                headerLabel.FadeTo(expand ? 1 : 0, 200);
+                this.FadeTo(expand ? 1 : 0);
```

Comment:
https://github.com/ppy/osu/blob/4e85481fd417131c2fd88f288e1ef95624358ae7/osu.Game/Overlays/Mods/ModSection.cs#L54-L56
That can be simplified into `this.FadeTo(value.Any() ? 1 : 0, 200)`, let me explain why, because if you change the alpha value of a container, it will affect its children, and `headerLabel` is a child of this container

Reply:
But if use `this.FadeTo(value.Any() ? 1 : 0, 200)` instead, `FillFlowContainer`'s `LayoutEasing` will be trigger on 200 ms
.
or maybe use `this.FadeTo(value.Any() ? 1 : 0,value.Any() ? 200 : 0)` ?

Reply:
we want it to fade in and fade out on 200ms, to give a good look when the mod category shows,  unless there will be something wrong?

Reply:
https://streamable.com/35f5g
this is the effect when just use `this.FadeTo(value.Any() ? 1 : 0, 200);`